packages/sdk/src/single.ts

```diff
@@ -363,7 +376,7 @@ export class RegionRatelimit extends Ratelimit<RegionContext> {
       const [remaining, reset] = (await ctx.redis.eval(
         script,
         [key],
-        [maxTokens, intervalDuration, refillRate, now],
+        [maxTokens, intervalDuration, refillRate, now]
```

Comment:
I couldn't leave a comment there, so I am adding this to here: The value of the key is wrong, a bucket should not be associated with a time window. So, the key should be equal to just the identifier

Reply:
I didn't realise the ts sdk has this problem, will fix.